### ISSUE TITLE
Something went wrong on login

### ISSUE THREAD

**Issue by chiragpurohit71085** [a85/POSTMan-Chrome-Extension#4713]
Tried on web as well as app, it suddenly said session time out and now I am not able to login.

**Comment by chiragpurohit71085** [comment]
![2018_06_17_10_01_13](https://user-images.githubusercontent.com/15859081/41504706-7d16209e-7215-11e8-94a9-17f2745755af.jpg)

**Comment by chiragpurohit71085** [comment]
![2018_06_17_10_02_48](https://user-images.githubusercontent.com/15859081/41504711-9d4d0382-7215-11e8-9773-16c36beb2d7d.jpg)

**Comment by adan92** [comment]
I have the same problem in web and app.

**Comment by adan92** [comment]
<img width="939" alt="2018-06-16 23_46_37-devtools - identity getpostman com_login" src="https://user-images.githubusercontent.com/15823844/41504783-8e0bfb0e-71bf-11e8-8ab1-c228341c5dcb.png">

**Comment by thecuriousdev** [comment]
@chiragpurohit71085 @adan92 are you still facing the problem?

**Comment by carmanchris31** [comment]
I was just experiencing this too, I think. I thought I had changed my password and didn't think to check the network logs to see if it was a 500. I just tried again to confirm the network call and was able to login successfully. Seems like this may have been resolved?

EDIT: I was able to log into the app as well at which point I immediately got the session expired message again and had to log in once more. All is well now, thanks!
